src/modules/mc_pos_control/mc_pos_control_params.c

```diff
@@ -469,7 +481,31 @@ PARAM_DEFINE_FLOAT(MPC_ACC_UP_MAX, 5.0f);
  * @decimal 2
  * @group Multicopter Position Control
  */
-PARAM_DEFINE_FLOAT(MPC_ACC_DOWN_MAX, 5.0f);
+PARAM_DEFINE_FLOAT(MPC_ACC_DOWN_MAX, 2.0f);
+
+/**
+ * Maximum jerk in manual controlled mode for braking to zero
+ *
+ * @unit m/s/s/s
+ * @min 2.0
+ * @max 15.0
+ * @increment 1
+ * @decimal 2
+ * @group Multicopter Position Control
+ */
+PARAM_DEFINE_FLOAT(MPC_JERK_MAX, 5.0f);
```

Comment:
do you want toenable all smoothing features by default?
there are two sides to it:
- + the normal user who has a well supported airframe that just works probably wants smoothing
- + the developer can see what values make sense
- - the hobby user or developer that wants to tune the controller (which happens a lot I guess) has to study all these parameters to diable smoothing first. I myself currently don't remember all of them.

A nice solution would be one parameter that disables it all (e.g. set jerk to 0 or what not) or at least a guide that tells you exactly which steps to take...